send-transaction-service/src/transaction_client.rs

```diff
@@ -227,37 +227,15 @@ where
 /// * Update the validator identity keypair and propagate the changes to the
 ///   scheduler. Most of the complexity of this structure arises from this
 ///   functionality.
+#[derive(Clone)]
 pub struct TpuClientNextClient {
     runtime_handle: Handle,
     sender: mpsc::Sender<TransactionBatch>,
-    bind_socket: UdpSocket,
-    // This handle is needed to implement `NotifyKeyUpdate` trait. It's only
-    // method takes &self and thus we need to wrap with Mutex.
-    join_and_cancel: Arc<Mutex<(Option<TpuClientJoinHandle>, CancellationToken)>>,
-    leader_forward_count: u64,
+    update_certificate_sender: watch::Sender<Option<StakeIdentity>>,
+    //handle: TpuClientJoinHandle,
```

Comment:
This client is cloned inside the SendTransctionService because it is used in two thread: one that initially sends the transactions and the one which retries sending failed. Because I need Clone, I cannot have handle there. But I'm not sure that I really need this handle. @ilya-bobyr what do you think?

Reply:
It seems that in the previous version you had the join handle inside an `Arc<Mutex<>>` allowing for the struct to be copied.
I also do not know if you need it.
It would allow you to wait for the broadcaster to shut down, if you want to.
Other than that, it does not seem to be useful.